crawl_tool/crawl.py

```diff
@@ -0,0 +1,158 @@
+"""Crawl publications from ECMWF homepage."""
+
+import os
+import re
+import time
+import urllib.request
+
+import click
```

Comment:
a small question, why use `click` instead of the common `argparse`

Reply:
https://click.palletsprojects.com/en/8.1.x/why/#why-not-argparse
- argparse has built-in behavior to guess if something is an argument or an option. This becomes a problem when dealing with incomplete command lines; the behaviour becomes unpredictable without full knowledge of a command line. This goes against Click’s ambitions of dispatching to subparsers.
- argparse does not support disabling interspersed arguments. Without this feature, it’s not possible to safely implement Click’s nested parsing.